ci/travis/install-dependencies.sh

```diff
@@ -309,6 +309,11 @@ install_dependencies() {
     pip install -r "${WORKSPACE_DIR}"/python/requirements_tune.txt
   fi
 
+  # Additional dependnecy for building docker images.
+  if [ "${LINUX_WHEELS-}" = 1 ]; then
+    pip install docker
```

Comment:
let's remove this? because this doesn't install the wheel in the right conda env. The env we are running in this script is overridden by test-wheel.sh

Reply:
Sounds good!